packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/simulationservice/CalibrationRequestCiemss.java

```diff
@@ -15,13 +18,16 @@
 // Used to kick off a calibration job in simulation-service
 public class CalibrationRequestCiemss implements Serializable {
 	@JsonAlias("model_config_id")
-	private String modelConfigId;
+	private UUID modelConfigId;
 
 	private Object extra;
```

Comment:
We had serialization problems with Object before, should we switch to JsonNode while we are in the area? @dvince2 ?

Reply:
The issue there was for serialization into postgres, which I don't believe these are.  However if `extra` is just a json blob we should probably just change it to JsonNode since thats what it is, and it will potentially save us a headace in the future